packages/riipen-ui/src/components/Table.test.jsx

```diff
@@ -0,0 +1,129 @@
+import { mount } from "enzyme";
+import toJson from "enzyme-to-json";
+import React from "react";
+
+import defaultTheme from "../themes/default";
+
+import Table from "./Table";
+import TableHeader from "./TableHeader";
+
+describe("<Table>", () => {
+  it("renders without errors", () => {
+    let error;
+
+    try {
+      mount(<Table />);
+    } catch (e) {
+      error = e;
+    }
+
+    expect(error).toEqual(undefined);
+  });
+
+  it("renders correct snapshot", () => {
+    const wrapper = mount(<Table />);
+
+    expect(toJson(wrapper)).toMatchSnapshot();
+  });
+
+  describe("default props", () => {
+    it("sets correct default props", () => {
+      const wrapper = mount(<Table />);
+
+      const component = wrapper.find("Table");
+      expect(component.props().backgroundColor).toEqual("grey50");
+      expect(component.props().layout).toEqual("auto");
+    });
+  });
+
+  describe("backgroundColor prop", () => {
+    it("sets custom valid backgroundColor with mapped hex color", () => {
+      const color = "grey400";
+
+      const wrapper = mount(<Table backgroundColor={color} />);
+
+      expect(wrapper.find("JSXStyle").props().dynamic).toContain(
+        `${defaultTheme.palette.grey[400]}`
+      );
+    });
+
+    it("sets transparent color given transparent backgroundColor", () => {
```

Comment:
The grey and transparent colors are two difference oneOf types for backgroundColor, so it doesn't seem like this second test is necessary.

Reply:
Ya I agree. We are testing the same thing twice here.